### PR TITLE
Adding timeout so full DFU works seamlessly

### PR DESCRIPTION
**Pull request by mjdietzx**
Currently there is a timeout in writeMode after the device disconnects and in connect() after we call device.gatt.connect() and before discovering services. With these timeouts DFU works seamlessly.

Also made constant variables naming styles more consistent.

### CODE REVIEW

File: dist/dfu.js [hunk 1]
```diff
@@ -46,33 +46,23 @@
 
     var LITTLE_ENDIAN = true;
     
-    var packetSize = 20;
-    var notifySteps = 40;
+    var PACKET_SIZE = 20;
```
Comment (thegecko) [inline dist/dfu.js:49]: "Global" variables like this aren't usually capitalized in JavaScript, that's more a 'C' const thing.

File: dist/dfu.js [hunk 2]
```diff
@@ -130,10 +130,14 @@
...
             .then(function() {
                 log("writing modeData");
-                return characteristics.controlChar.writeValue(new Uint8Array([OPCODE.START_DFU, ImageType.Application]));
+                return characteristics.controlChar.writeValue(new Uint8Array([Opcode.START_DFU, ImageType.APPLICATION]));
             })
             .then(function() {
                 log("modeData written");
+                setTimeout(function() {
+                    resolve(device); // TODO: Remove this when gattserverdisconnected is implemented.
+                }, 1e3);
```
Comment (thegecko) [inline dist/dfu.js:139]: Please use milliseconds instead of '1e3'

**Comment by thegecko** [comment]
@mjdietzx Can you split apart the variable formatting into a separate PR, it's not relevant to the timeout.

**Comment by mjdietzx** [comment]
Yes, I will make a new PR with only the timeout fixes.

**Pull request closed without merging**